schedview/app/scheduler_dashboard/scheduler_dashboard.py

```diff
@@ -1449,6 +1463,54 @@ def __init__(self, data_dir=None):
             self.param["scheduler_fname"].update(path=f"{data_dir}/*scheduler*.p*")
 
 
+class USDFScheduler(Scheduler):
```

Comment:
We should call it something like `LFAScheduler` rather than `USDFScheduler` (and we should consider renaming all of these to something less ambiguous, like `LFASchedulerSnapshotDashboard`, so it doesn't get confused with the actual scheduler, or other tools relate to the scheduler). The LFA used might not always be at the USDF (e.g. it could be at the summit), and we may use other access methods at the USDF.